libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -165,6 +172,25 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     );
   }
 
+  ngAfterViewChecked() {
+    if (this._hasYearNavigator) {
+      this.constrainDropdownPopoutHeight();
+    }
+  }
+
+  private constrainDropdownPopoutHeight(): void {
+    const tableHeightUnitless = this.element.querySelector('table').offsetHeight;
```

Comment:
I think the main point is that they are unitless values so I'd prefer to not use `px` in their name. I think `tableHeight` and `itemHeight` are good names.

I'm not sure if the linter is configured to warn against explicit type annotation when it can be inferred, but otherwise I think it's a good suggestion to annotate them as `number` explicitly.